src/Resources/views/dashboard.html.twig

```diff
@@ -58,11 +58,47 @@
                         <td><a class="btn btn-danger" href="{{ path('karo-io.reject_failed_message', {id: failedMessage.id}) }}">Reject</a></td>
                     </tr>
                 {% else %}
-                    <tr><td colspan="6">Your failure transport is empty. You're killing it!</td></tr>
+                    <tr>
+                        <td colspan="6">Your failure transport is empty. You're killing it!</td>
+                    </tr>
                 {% endfor %}
             </table>
         {% endif %}
     {% endif %}
+
+    {% if statistics.metrics|length %}
+        <h2>Statistics for last 24 hours</h2>
+
+        <h3 class="mt-3">Overall statistics</h3>
+        <div class="card" style="width: 18rem;">
+            <ul class="list-group list-group-flush">
+                <li class="list-group-item">Messages count: {{ statistics.messagesCount }}</li>
+                <li class="list-group-item">Messages per hour: {{ statistics.messagesHandledPerHour }}</li>
+                <li class="list-group-item">Average waiting time: {{ statistics.averageWaitingTime }}</li>
+                <li class="list-group-item">Average handling time: {{ statistics.averageHandlingTime }}</li>
+            </ul>
+        </div>
+
+        <h3 class="mt-3">Per message type statistics</h3>
+        <table class="table">
+            <tr>
+                <th>Message class</th>
+                <th>Messages count</th>
+                <th>Messages per hour</th>
+                <th>Average waiting time</th>
+                <th>Average handling time</th>
+            </tr>
+            {% for metrics in statistics.metrics %}
+                <tr>
+                    <td>{{ metrics.class }}</td>
+                    <td>{{ metrics.messagesCount }}</td>
+                    <td>{{ metrics.messagesHandledPerHour }}</td>
+                    <td>{{ metrics.averageWaitingTime }}</td>
+                    <td>{{ metrics.averageHandlingTime }}</td>
+                </tr>
+            {% endfor %}
+        </table>
+    {% endif %}
```

Comment:
Obviously, the BIG todo will be to make this admin dashboard really sing. I'll talk to Leanna about it next week.

Reply:
yep, sorry i can't help on this :/